google/cloud/aiplatform/metadata/metadata.py

```diff
@@ -111,5 +203,50 @@ def _validate_experiment_and_run(self, method_name: str):
                 f"No run set. Make sure to call aiplatform.set_run('my-run') before trying to {method_name}. "
             )
 
+    def _query_runs_to_data_frame(
+        self, context_name: str, context_resource_name: str, source: str
+    ) -> pd.DataFrame:
+        filter = f'schema_title="{constants.SYSTEM_RUN}" AND in_context("{context_resource_name}")'
+        run_executions = _Execution.list(filter=filter)
+
+        context_dict = []
```

Comment:
dict or array?

Reply:
renamed this to context_summary to avoid the confusion.